app/_src/networking/transparent-proxy/kubernetes.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Transparent Proxy on Kubernetes
+content_type: how-to
+---
+
+{% assign docs = "/docs/" | append: page.version %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma-system = site.mesh_namespace %}
+{% assign controlPlane = site.set_flag_values_prefix | append: "controlPlane" %}
+{% assign transparentProxy = site.set_flag_values_prefix | append: "transparentProxy" %}
+{% assign kuma-control-plane = site.mesh_cp_name %}
+{% assign kuma-control-plane-config = kuma-control-plane | append: "-config" %}
+
+In Kubernetes mode, transparent proxy is automatically set up through the `kuma-init` container or [Kuma CNI]({{ docs }}/production/dp-config/cni/). By default, it intercepts all incoming and outgoing traffic and routes it through the `kuma-dp` sidecar container, so no changes to the application code are needed.
+
+{{ Kuma }} works smoothly with [Kubernetes DNS for Services and Pods](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) and provides its own [Kuma DNS]({{ docs }}/networking/dns/), which is especially helpful in multi-zone setups for cross-zone service discovery.
+
+In this mode, {{ Kuma }} requires the transparent proxy to be enabled, so it **cannot be turned off**.
+
+## Configuration
+
+The default configuration works well for most scenarios, but there are cases where adjustments are needed.
+
+In Kubernetes mode, {{ Kuma }} there are three methods to adjust the configuration. Each can be used on its own or combined with others if needed.
+
+{% warning %}
+It’s best to stick to one method whenever possible. Using more than one can make things more complicated and harder to troubleshoot, as it may not be clear where each setting comes from. If you need to combine methods, check the [**Order of Precedence** section](#order-of-precedence) to see what the final configuration will look like based on the priority of each setting.
+{% endwarning %}
+
+<!-- vale Google.Headings = NO -->
+### Control Plane Runtime Configuration
+<!-- vale Google.Headings = YES -->
+
+The control plane runtime configuration is a section of {{ Kuma }}'s configuration that contains important settings for how the control plane operates, including options for the transparent proxy and other key components.
+
+Some transparent proxy settings can be adjusted here, and **for certain settings, this is the only place they can be modified**.
+
+Currently, it’s best to use the control plane runtime configuration as the main place to store transparent proxy settings that apply to all [workloads](https://kubernetes.io/docs/concepts/workloads/). In the future, once the [Configuration in ConfigMap](#configuration-in-configmap-experimental) feature is fully supported, we’ll recommend using ConfigMaps for these settings. However, a few restricted settings that are rarely customized will still need to be in the control plane’s runtime configuration.
```

Comment:
```suggestion
Currently, it’s best to use the control plane runtime configuration as the main place to store transparent proxy settings that apply to all [workloads](https://kubernetes.io/docs/concepts/workloads/).
```

I think we should only be talking about current state of project in docs. This should be a snapshot of knowledge around specific Kuma version. This might happen, but we could also switch our approach and remove config maps

Reply:
addressed